packages/storefront-sdk/package.json

```
@@ -0,0 +1,61 @@
+{
+	"name": "@nacelle/storefront-sdk-v2",
+	"version": "2.0.0",
+	"type": "module",
+	"author": "Nacelle Inc.",
+	"license": "Apache-2.0",
+	"files": [
+		"src",
+		"dist"
+	],
+	"scripts": {
+		"dev": "vite",
+		"build": "vite build && tsc --project tsconfig.production.json && mv dist/src dist/types",
+		"preview": "vite preview",
+		"precommit": "lint-staged",
+		"test": "vitest",
+		"coverage": "vitest run --coverage"
+	},
+	"homepage": "https://github.com/getnacelle/nacelle-js/tree/main/packages/sorefront-sdk#readme",
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/getnacelle/sorefront-sdk.git"
+	},
+	"bugs": {
+		"url": "https://github.com/getnacelle/sorefront-sdk/issues"
+	},
+	"engines": {
+		"node": ">=14.14 <19",
```

1. Let's choose a higher Node version for the minimum. Node 14 will be deprecated in April 2023, and there are language features that we'd like to have the freedom to use that are not available in Node 14. 
2. Removing the upper bound could alleviate the future maintenance burden.

```suggestion
		"node": ">=16.11",
```